### PR TITLE
Post-release bug 2023.12.2

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adds a post-release bug. WILL MERGE AFTER RELEASE HITS ALL CELLS IN PROD.
- **Is this PR related to a Monolith release?** Yes. 2023.12.2

### Resolves:

* [OKTA-673562](https://oktainc.atlassian.net/browse/OKTA-673562)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/f6c4945171c80b72bb0905361a12e0f92534fc95/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 529 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2e9df268-d2ee-49b8-aa05-d3a1643f34d6) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/f6c4945171c80b72bb0905361a12e0f92534fc95/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 369 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e4367e27-b941-47a3-902d-363a7a0ef80b) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.f8a78211-c394-423b-8456-b25fa418d716?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ title: Okta Identity Engine API Products release notes 2023
 
 * When using [Okta Expression Language in Identity Engine](/docs/reference/okta-expression-language-in-identity-engine/#group-functions), the `EXACT` and `STARTS_WITH` operators couldn't be used with the `group.profile.name` key to return exact matches. (OKTA-636560)
 
+* Some fre trial orgs could send customized email templates. (OKTA-673562)
```
Comment (MarkCowan-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:25]:
```suggestion
* Some free-trial orgs could send customized email templates. (OKTA-673562)
```

Comment (MarkCowan-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:25]:
```suggestion
* Some free-trial orgs could send customized email templates. (OKTA-673562)
```

**Review by MarkCowan-okta — APPROVED** [review]
Hi Brent. Typo and hyphen nit, otherwise good to go.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/cd2ad33fc843d6a6a281c6e7feb59d689253fc56/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bae82997-f90b-4441-96f0-e04a633cef43) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/cd2ad33fc843d6a6a281c6e7feb59d689253fc56/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 369 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/92f87b3e-c49c-4d2e-bb2f-b9c2a104b213) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.33319fc9-8b29-4e67-b7bd-dad226232444?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/cd2ad33fc843d6a6a281c6e7feb59d689253fc56/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bae82997-f90b-4441-96f0-e04a633cef43) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/cd2ad33fc843d6a6a281c6e7feb59d689253fc56/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 369 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/92f87b3e-c49c-4d2e-bb2f-b9c2a104b213) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.33319fc9-8b29-4e67-b7bd-dad226232444?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/cd2ad33fc843d6a6a281c6e7feb59d689253fc56/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bae82997-f90b-4441-96f0-e04a633cef43) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/cd2ad33fc843d6a6a281c6e7feb59d689253fc56/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 369 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/92f87b3e-c49c-4d2e-bb2f-b9c2a104b213) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.33319fc9-8b29-4e67-b7bd-dad226232444?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/bce08dd27aa0b0f0c457376df6c5aface2a10a17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/89b43296-b3d3-4a40-bbbf-6d613050eb54) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/bce08dd27aa0b0f0c457376df6c5aface2a10a17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/36e94bd6-a12b-4d29-8236-0acc74bb440e) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e7aa3e00-186d-4863-821a-bb19a726a6b5?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by grahamsmith-okta — APPROVED** [review]
re-approving

**Pull request merged**